_data/schedule.yml

```diff
@@ -71,11 +71,18 @@
   day3: true
   time: 10:15-11:00
 
+# Poster Session
+Thursday, February 15 starting at 3:15pm
```

Comment:
Needs to be commented out or site doesn't build

Reply:
Doh! Forgot about that.